libs/openFrameworks/graphics/ofBitmapFont.h

```diff
@@ -22,7 +23,7 @@ class ofBitmapFont{
 	~ofBitmapFont();
 	ofMesh getMesh(const string & text, int x, int y, ofDrawBitmapMode mode=OF_BITMAPMODE_MODEL_BILLBOARD, bool vFlipped=true) const;
 	const ofTexture & getTexture() const;
-	ofRectangle getBoundingBox(const string & text, int x, int y) const;
+	ofRectangle getBoundingBox(const string & text, int x, int y, ofDrawBitmapMode mode = ofGetStyle().drawBitmapMode, bool vFlipped = ofIsVFlipped()) const;
```

Comment:
Ha, right. There seems to be no other way to get the current bitmap mode. And right, it also looked funny to me, but it is there to keep backwards compatibility.